cmd/vela-docker/main.go

```diff
@@ -20,8 +20,8 @@ func main() {
 
```

Comment:
I believe the name of the directory containing this file needs to be renamed 👍 

Currently it's:

`cmd/vela-docker/main.go`

When I believe it should be:

`cmd/vela-kaniko/main.go`